src/winter_onboarding_2021/4clojure/alisha.clj

```diff
@@ -4,4 +4,6 @@
 
 (defn problem5 [] (conj '(2 3 4) 1))
 
-(defn problem6 [] (list :a :b :c) (vec '(:a :b :c)) (vector :a :b :c))
\ No newline at end of file
+(defn problem6 [] (list :a :b :c) (vec '(:a :b :c)) (vector :a :b :c))
```

Comment:
Check out [clojure style guide](https://github.com/bbatsov/clojure-style-guide) and try to follow it.